### PR TITLE
Refactor TestReorderJoins

### PR DESCRIPTION
**Pull request by kokosing**

### CODE REVIEW

File: presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestReorderJoins.java [hunk 1]
```diff
@@ -14,34 +14,34 @@
...
 import org.testng.annotations.Test;
 
-import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.tableScan;
 import static org.testng.Assert.assertEquals;
 
 public class TestReorderJoins
         extends BasePlanTest
 {
-    private static final PlanMatchPattern SUPPLIER_TABLESCAN = tableScan("supplier", ImmutableMap.of("S_SUPPKEY", "suppkey", "S_NATIONKEY", "nationkey"));
-    private static final PlanMatchPattern PART_TABLESCAN = tableScan("part", ImmutableMap.of("P_PARTKEY", "partkey"));
```
Comment (findepi) [inline presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestReorderJoins.java:33]: those removals should be part of "Refactor q2 test to check only join ordering" commit, right?

File: presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestReorderJoins.java [hunk 2]
```diff
@@ -33,7 +33,9 @@
 {
     public TestReorderJoins()
     {
-        super(ImmutableMap.of(SystemSessionProperties.JOIN_REORDERING_STRATEGY, "COST_BASED"));
+        super(ImmutableMap.of(
+                SystemSessionProperties.JOIN_REORDERING_STRATEGY, "COST_BASED",
+                SystemSessionProperties.JOIN_DISTRIBUTION_TYPE, "AUTOMATIC"));
```
Comment (findepi) [inline presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestReorderJoins.java:38]: `"automatic"`. However, I'd prefer not to have this change.

Comment (losipiuk) [reply presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestReorderJoins.java:38]: As talked. Default appears to be REPARTITIONED. So we need ths.

Comment (losipiuk) [reply presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestReorderJoins.java:38]: If "automatic" then also "cost_based".
But I am not sure if that can be lowercase for enums. Can it?

**Comment by losipiuk** [comment]
@kokosing that is merged in right?

**Comment by losipiuk** [comment]
I see it is

**Pull request closed without merging**